lib/git/worktree_test.go

```diff
@@ -0,0 +1,60 @@
+package git
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/enfabrica/enkit/lib/errdiff"
+
+	"github.com/prashantv/gostub"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewTempWorktree(t *testing.T) {
+	testCases := []struct {
+		desc         string
+		committish   string
+		wantCmds     []*exec.Cmd
+		wantErr      string
+		wantCloseErr string
+	}{
+		{
+			desc:       "successfully creates temp worktree",
+			committish: "some_branch_name",
+			wantCmds: []*exec.Cmd{
+				&exec.Cmd{
+					Path: "git",
+					Args: []string{"worktree", "add", "foo", "some_branch_name"},
+					Dir:  "/foo/bar",
+				},
+				&exec.Cmd{
+					Path: "git",
+					Args: []string{"worktree", "remove", "foo"},
+					Dir:  "/foo/bar",
+				},
+			},
+		},
+	}
```

Comment:
not objecting, but there is a single testcase here. Is it worth the complexity of having this struct/array? makes the test arguably much harder to read and understand.

Reply:
Yeah, this is nonsensical; removed the test table.